InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -228,29 +249,34 @@ public void spinUp() {
     }
 
     public void stopSpinningUp() {
-        // Spin up the feeder.
+        // Stop the feeder.
         feeder.set(0);
         SmartDashboard.putString(getName() + "/Feeder State", "Doing Nothing");
 
-        // Spin up the shooter.
+        // Stop the shooter.
         ballPropulsionMotor.set(0);
         SmartDashboard.putString(getName() + "/Shooter State", "Doing Nothing");
 
+        // Stop spinning up.

```

Comment:
Can we put something here that this makes sure the periodic() function doesn't try and spinup the shooter or feeder anymore?

Reply:
It does that already, that's the whole point of the stop spinning up function.
Unless you're saying I should put a comment here that says that, in which case sure.

Reply:
Yeah, sorry, I meant add a comment explaining that. I worded it poorly.